### [CVE-2023-39661](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-39661)
![](https://img.shields.io/static/v1?label=Product&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Version&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Vulnerability&message=n%2Fa&color=brighgreen)

### Description

An issue in pandas-ai v.0.9.1 and before allows a remote attacker to execute arbitrary code via the _is_jailbreak function.

### POC

#### Reference
- https://github.com/gventuri/pandas-ai/issues/410

#### Github
No PoCs found on GitHub currently.

